Command/DoctrineCommand.php

```diff
@@ -60,6 +61,10 @@ protected function getEntityManager($name, $shardId = null)
         $manager = $this->getDoctrine()->getManager($name);
 
         if ($shardId) {
+            if (! $manager instanceof EntityManagerInterface) {
+                throw new LogicException(sprintf('Sharding is supported only in EntityManager of instance "%s".', EntityManagerInterface::class));
```

Comment:
This looks weird… I mean all `EntityManagers` implement `EntityManagerInterface`, right? Should that sentence mention `ObjectManager` or something like that instead?

Reply:
`getConnection` doesn't exist in ObjectManager, only EntityManagerInterface